src/main/java/net/minecraftforge/common/ForgeModContainer.java

```diff
@@ -278,22 +278,22 @@ private static void syncConfig(boolean load)
 
         // Client-Side only properties
         propOrder = new ArrayList<String>();
-        prop = config.get(Configuration.CATEGORY_CLIENT, "replaceVanillaBucketModel", Boolean.FALSE,
+        prop = config.get(Configuration.CATEGORY_CLIENT, "replaceVanillaBucketModel", false,
                 "Replace the vanilla bucket models with Forges own dynamic bucket model. Unifies bucket visuals if a mod uses the Forge bucket model.");
         prop.setLanguageKey("forge.configgui.replaceBuckets").setRequiresMcRestart(true);
-        replaceVanillaBucketModel = prop.getBoolean(Boolean.FALSE);
+        replaceVanillaBucketModel = prop.getBoolean(false);
         propOrder.add(prop.getName());
 
-        prop = config.get(Configuration.CATEGORY_CLIENT, "java8Reminder", java8Reminder,
+        prop = config.get(Configuration.CATEGORY_CLIENT, "java8Reminder", 0,
                 "The timestamp of the last reminder to update to Java 8 in number of milliseconds since January 1, 1970, 00:00:00 GMT. Nag will show only once every 24 hours. To disable it set this to some really high number.");
-        java8Reminder = prop.getLong(java8Reminder);
+        java8Reminder = prop.getLong(0);
         prop.setLanguageKey("forge.configgui.java8Reminder");
         propOrder.add(prop.getName());
 
-        prop = config.get(Configuration.CATEGORY_CLIENT, "disableStairSlabCulling", disableStairSlabCulling,
+        prop = config.get(Configuration.CATEGORY_CLIENT, "disableStairSlabCulling", false,
                 "Disable culling of hidden faces next to stairs and slabs. Causes extra rendering, but may fix some resource packs that exploit this vanilla mechanic.");
-        disableStairSlabCulling = prop.getBoolean(disableStairSlabCulling);
-        prop.setLanguageKey("forge.configgui.disableStairSlabCulling").setRequiresMcRestart(false);
```

Comment:
But this makes the "reset to defaults" button not work, since it thinks the current value is the default.